src/civ2.js

```diff
@@ -52,183 +65,148 @@ module.exports = function(robot) {
+      const status = await civ2.getBranchInformation(branchName, msg.message.user.name);
+      msg.reply(`${message}\n${status}`);
+    })
+  );
+
+  robot.hear(
+    /merge pull requests (\S*)/,
+    responderFactory(async (msg) => {
+      const branchName = msg.match[1];
+      msg.reply(`Merging PRs for branch ${branchName}...`);
       const message = await civ2.mergePRs(branchName, msg.message.user.name);
       msg.reply(message);
-    } catch (err) {
-      respondToError(err, msg);
-    }
-  });
-
-  robot.hear(/close pull requests (\S*)/, async (msg) => {
-    const branchName = msg.match[1];
-    msg.reply(`Closing PRs for branch ${branchName}...`);
-
-    try {
+    })
+  );
+
+  robot.hear(
+    /close pull requests (\S*)/,
+    responderFactory(async (msg) => {
+      const branchName = msg.match[1];
+      msg.reply(`Closing PRs for branch ${branchName}...`);
       const message = await civ2.closePRs(branchName, msg.message.user.name);
       msg.reply(message);
-    } catch (err) {
-      respondToError(err, msg);
-    }
-  });
-
-  robot.hear(/delete branch (\S*)/, async (msg) => {
-    const branchName = msg.match[1];
-    msg.reply(`Deleting branch ${branchName}...`);
-
-    try {
+    })
+  );
+
+  robot.hear(
+    /delete branch (\S*)/,
+    responderFactory(async (msg) => {
+      const branchName = msg.match[1];
+      msg.reply(`Deleting branch ${branchName}...`);
       const message = await civ2.deleteBranches(branchName, msg.message.user.name);
       msg.reply(message);
-    } catch (err) {
-      respondToError(err, msg);
-    }
-  });
-
-  robot.hear(/update links (\S*)/, async (msg) => {
-    const branchName = msg.match[1];
-    msg.reply(`Updating lonks for branch ${branchName}...`);
-
-    try {
+    })
+  );
+
+  robot.hear(
+    /update links (\S*)/,
+    responderFactory(async (msg) => {
+      const branchName = msg.match[1];
+      msg.reply(`Updating links for branch ${branchName}...`);
       const message = await civ2.updatePRsDescriptions(branchName, msg.message.user.name);
       msg.reply(message);
-    } catch (err) {
-      respondToError(err, msg);
-    }
-  });
+    })
+  );
 
-  robot.hear(/update instance (\S*)( on (\S*) environment)?( to version (\S*))?/, async (msg) => {
-    const [, instance, , env, , version] = msg.match;
-    try {
+  robot.hear(
+    /update instance (\S*)( on (\S*) environment)?( to version (\S*))?/,
+    responderFactory(async (msg) => {
+      const [, instance, , env, , version] = msg.match;
       const targetVersion = await civ2.updateInstance(instance, env, version);
       msg.reply(`Instance <${instance}|${instance}> on "${env}" is updating to version "${targetVersion}"`);
-    } catch (err) {
-      respondToError(err, msg);
-    }
-  });
-
-  robot.hear(/Publicly release (\S*)/, async (msg) => {
-    const releaseSource = msg.match[1];
-    const messages = [];
-    try {
+    })
+  );
+
+  robot.hear(
+    /Publicly release (\S*)/,
```

Comment:
Why the leading capital? All other messages are lowercase. That flag `i` will solve that, but the regexps should all be in lowercase, for consistency.